mp/src/game/shared/momentum/weapon/weapon_mom_stickybomblauncher.cpp

```diff
@@ -0,0 +1,436 @@
+#include "cbase.h"
+
+#include "in_buttons.h"
+#include "mom_player_shared.h"
+#include "mom_stickybomb.h"
+#include "mom_system_gamemode.h"
+#include "weapon_mom_stickybomblauncher.h"
+
+#ifdef CLIENT_DLL
+#include "prediction.h"
+#else
+#include "momentum/mom_triggers.h"
+#endif
+
+#include "tier0/memdbgon.h"
+
+#define MOM_STICKYBOMB_MIN_CHARGE_VEL 900
+#define MOM_STICKYBOMB_MAX_CHARGE_VEL 2400
+#define MOM_STICKYBOMB_MAX_CHARGE_TIME 4.0f
+#define MOM_WEAPON_STICKYBOMB_COUNT 3
+
+IMPLEMENT_NETWORKCLASS_ALIASED(MomentumStickybombLauncher, DT_MomentumStickybombLauncher)
+
+BEGIN_NETWORK_TABLE_NOBASE(CMomentumStickybombLauncher, DT_MomentumStickybombLauncherLocalData)
```

Comment:
A while ago I was struggling a lot with some bugs concerning the stickybomb count variable and this is the way TF2 implemented the network tables so I had done it the same way to avoid any potential errors. It seems that removing the nobase table doesn't break anything so I can remove it.